_progress-bar.scss

```diff
@@ -0,0 +1,252 @@
+// progress-bar
+// ===
+
+// Configurable Variables
+// ---
+
+$progress-bar__progress-color: black !default;
+$progress-bar__success-color: $success-color !default;
+$progress-bar__error-color: $error-color !default;
+$progress-bar__background-color: $neutral-50 !default;
+
+
+// Dependencies
+// ---
+
+// Override Font Awesome's default path
+$fa-font-path: '/bower_components/font-awesome/fonts';
+
+@import 'stencil-variables/variables';
+@import 'level/level';
+@import 'font-awesome/scss/font-awesome';
+@import 'stencil-utils/dist/visibility/visibility';
+
+
+// Progress Bar root
+// ---
+
+.c-progress-bar {
+    display: flex;
+    align-items: center;
+    height: $tap-size;
+    margin: 0 15px;
+}
+
+
+// Radial Process Bar
+// ---
+
+.c-progress-bar__radial {
+    &:not(:empty) {
+        position: relative;
+
+        width: 45px;
+        height: 45px;
+        margin-right: 10px;
+    }
+
+    .c-progress-bar__radial-background {
+        stroke: $progress-bar__background-color;
+    }
+
+    .c-progress-bar__text {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+
+        transform: translate(-50%, -50%);
+    }
+
+    .c--progress & {
+        .c-progress-bar__radial-progress {
+            stroke: $progress-bar__progress-color;
+        }
+    }
+
+    .c--success & {
+        .c-progress-bar__radial-progress {
+            stroke: $progress-bar__success-color;
+        }
+    }
+
+    .c--error & {
+        .c-progress-bar__radial-progress {
+            stroke: $progress-bar__error-color;
+        }
+    }
+
+    .c--direction-rtl & {
+        .c-progress-bar__svg {
+            transform: scaleX(-1);
+        }
+    }
+}
+
+
+// Linear Progress Bar
+// ---
+//
+// 1. When the radial progress bar is shown, we want to hide the bar but keep the text
+
+.c-progress-bar__linear {
```

Comment:
Good point. I've created a new element that holds the label and the linear progress bar so the `.c-progress-bar__linear` element is no longer included in the radial variation.